zkevm-circuits/src/evm_circuit.rs

```diff
@@ -112,20 +112,20 @@ impl<F: Field> EvmCircuit<F> {
         layouter: &mut impl Layouter<F>,
         block: &Block<F>,
     ) -> Result<(), Error> {
-        self.execution.assign_block(layouter, block, false)?;
+        self.execution.assign_block(layouter, block)?;
         Ok(())
     }
 
-    #[cfg(any(feature = "test", test))]
-    /// Assign exact steps in block without padding for unit test purpose
-    pub fn assign_block_exact(
-        &self,
-        layouter: &mut impl Layouter<F>,
-        block: &Block<F>,
-    ) -> Result<(), Error> {
-        self.execution.assign_block(layouter, block, true)?;
-        Ok(())
-    }
+    // #[cfg(any(feature = "test", test))]
+    // /// Assign exact steps in block without padding for unit test purpose
+    // pub fn assign_block_exact(
+    //     &self,
+    //     layouter: &mut impl Layouter<F>,
+    //     block: &Block<F>,
+    // ) -> Result<(), Error> {
+    //     self.execution.assign_block(layouter, block, true)?;
+    //     Ok(())
+    // }
```

Comment:
We should avoid checking in the commented-off code.
```suggestion
```

Reply:
Thanks for mentioning this! I forgot to remove it.